packages/web/src/scss/tools/__tests__/_responsive-properties.test.scss

```diff
@@ -0,0 +1,122 @@
+@use 'true' as test;
+@use '../responsive-properties';
+
+@include test.describe('responsive properties functions and mixins') {
+    @include test.it('should return correct property with custom property cascade') {
+        @include test.assert() {
+            @include test.output() {
+                .test {
+                    @include responsive-properties.create-cascade(
+                        $property: 'row-gap',
+                        $input-custom-property-base-name: 'gap',
```

Comment:
I shortened the base name because of Prettier (the new suffixes make generated lines longer). I also changed the property to `row-gap` so it cannot be confused with the base name.